application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -41,9 +44,22 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        // TODO A list might be better than comma separated, which is hard to read
-        event.replyEmbeds(MessageUtils.generateEmbed("All available tags",
-                String.join(", ", tagSystem.getAllIds()), event.getUser(), TagSystem.AMBIENT_COLOR))
+        int MAX_TAGS_THRESHOLD_WARNING = 200;
+        if (tagSystem.getAllIds().size() > MAX_TAGS_THRESHOLD_WARNING) {
+            // TODO Implement the edge case
+
+            logger.warn(
+                    "The amount of tags is very high and it might soon exceed the maximum character limit. The code should be adjusted to support this edge case soon.");
+        }
+        String tagListText = tagSystem.getAllIds()
+            .stream()
+            .sorted()
+            .map(tag -> "* " + tag)
```

Comment:
Actually, I propose a minor change for better UX. Replace the `*` by [U+2022 bullet point](https://www.compart.com/en/unicode/U+2022), i.e. `•`